src/python/WMComponent/JobCreator/JobCreatorWorker.py

```diff
@@ -258,28 +239,28 @@ def __call__(self, parameters):
                 # We'll have to just keep skipping it
                 wmTask = None
                 seederList = []
-                logging.error("Have no task for workflow %i" % (workflow.id))
-                logging.error("Aborting Subscription %i" % (subscriptionID))
+                logging.error("Have no task for workflow %i", workflow.id)
+                logging.error("Aborting Subscription %i", subscriptionID)
                 continue
 
             else:
                 wmTask = wmWorkload.getTaskByPath(workflow.task)
                 if hasattr(wmTask.data, 'seeders'):
-                    manager    = SeederManager(wmTask)
-                    seederList = manager.getSeederList()
+                    manager = GeneratorManager(wmTask)
```

Comment:
Hmm, I guess it was never used then(?)

Reply:
Quite possibly. Some things I tried to find out if they were used or not. Others I just tried to fix up